core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -359,9 +359,10 @@ private[log] class LogCleanerManager(val logDirs: Seq[File],
    *
    * @param dataDir                       The File object to be updated
    * @param partitionToUpdateOrAdd        The [TopicPartition, Long] map data to be updated. pass "none" if doing remove, not add
-   * @param topicPartitionToBeRemoved     The TopicPartition to be removed
+   * @param partitionToRemove             The TopicPartition to be removed
```

Comment:
Now that you are modifying this, probably it's worth mentioning that this `Option` parameter has a default value and what it means when `None` is passed.